docs/changelog/130427.yaml

```diff
@@ -1,5 +1,18 @@
 pr: 130427
-summary: Disallow brackets in unquoted index pattersn
+summary: Disallow brackets in unquoted index patterns
 area: ES|QL
-type: bug
-issues: []
+type: breaking
+issues:
+  - 130378
+breaking:
+  title: Unquoted index patterns do not allow `(` and `)` characters
+  area: ES|QL
+  details: >-
+    So far, ES|QL accepted unquoted index patterns containing brackets, eg. `FROM index(1) | ENRICH policy(2)`
+
+    This can potentially conflict with subquery syntax, that uses brackets as a delimiter.
+
+    From now on, using brackets is only allowed in quoted index patterns, ie. the query above is no longer valid,
+    while the following is valid: `FROM "index(1)" | ENRICH "policy(2)"`
```

Comment:
```suggestion
    Previously, ES|QL accepted unquoted index patterns containing brackets, such as `FROM index(1) | ENRICH policy(2)`.

    This query syntax is no longer valid because it could conflict with subquery syntax, where brackets are used as delimiters.

    Brackets are now only allowed in quoted index patterns. For example: `FROM "index(1)" | ENRICH "policy(2)"`.
```